contrib/screener-api/screener/screener.go

```diff
@@ -225,42 +226,32 @@ func (s *screenerImpl) authMiddleware(cfg config.Config) gin.HandlerFunc {
 		_, span := s.metrics.Tracer().Start(c.Request.Context(), "authMiddleware")
 		defer span.End()
 
-		appID := c.Request.Header.Get("AppID")
-		timestamp := c.Request.Header.Get("Timestamp")
-		nonce := c.Request.Header.Get("Nonce")
-		signature := c.Request.Header.Get("Signature")
-		queryString := c.Request.Header.Get("QueryString")
-		bodyBytes, _ := io.ReadAll(c.Request.Body)
-		bodyStr := string(bodyBytes)
-
-		c.Request.Body = io.NopCloser(strings.NewReader(bodyStr))
+		appID := c.Request.Header.Get("X-Signature-appid")
+		timestamp := c.Request.Header.Get("X-Signature-timestamp")
+		nonce := c.Request.Header.Get("X-Signature-nonce")
+		signature := c.Request.Header.Get("X-Signature-signature")
+		queryString := c.Request.URL.RawQuery
 
-		span.SetAttributes(
```

Comment:
would discourage this until we know what works unless we have an alternative way to see this

Reply:
I'd actually add a span that logs all the headers & body of the request so we can debug easily

Reply:
added